client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -201,21 +253,44 @@ function CheckoutSummaryPlanFeatures() {
 	);
 }
 
-function getPlanFeatures( plan, translate, hasDomainsInCart, hasRenewalInCart ) {
+function getPlanFeatures(
+	plan,
+	translate,
+	hasDomainsInCart,
+	hasRenewalInCart,
+	isMonthlyPricingTest
+) {
 	const showFreeDomainFeature = ! hasDomainsInCart && ! hasRenewalInCart;
-	if (
-		'personal-bundle' === plan.wpcom_meta?.product_slug ||
-		'personal-bundle-2y' === plan.wpcom_meta?.product_slug
-	) {
+	const productSlug = plan.wpcom_meta?.product_slug;
+	const annualPlansOnly = translate( '(annual plans only)', {
+		comment: 'Label attached to a feature',
+	} );
+
+	if ( ! productSlug ) {
+		return [];
+	}
+
+	if ( [ 'personal-bundle', 'personal-bundle-2y' ].includes( productSlug ) ) {
 		return [
+			isMonthlyPricingTest && translate( 'Live chat and email support' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Email and live chat support' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/monthly-pricing-abtest)